extensions/amp-story/1.0/media-pool.js

```diff
@@ -785,7 +785,7 @@ export class MediaPool {
           if (rewindToBeginning) {
             this.enqueueMediaElementTask_(
                 /** @type {!PoolBoundElementDef} */ (poolMediaEl),
-                new RewindTask());
+                new SetCurrentTimeTask());
```

Comment:
nit: seems more explicit to include the `0` than to fallback to the default

Reply:
You're right, it reads better. Thanks!